app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -61,4 +80,22 @@ public void showCredentials(String email, String pass) {
         textEmail.setText(email);
         textPass.setText(pass);
     }
+
+    @Override
+    public void goSignUp() {
+        Intent intent = new Intent(getContext(), SingUpActivity.class);
+        startActivity(intent);
+    }
+
+    @Override
+    public void goHome() {
+        Intent intent = new Intent(getContext(), HomeActivity.class);
+        startActivity(intent);
+    }
+
+    @Override
+    public void goTermsAndConditions() {
+        Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.terms_and_conditions)));
```

Comment:
Since it represents a link and not only a string to show, the terms and condition URL should not be a `String resource`, it should be a constant.